src/OpenTelemetry.Instrumentation.AWS/.publicApi/PublicAPI.Unshipped.txt

```diff
@@ -1,6 +1,12 @@
 #nullable enable
+OpenTelemetry.Instrumentation.AWS.SemanticConventionVersion
+OpenTelemetry.Instrumentation.AWS.SemanticConventionVersion.Latest = 0 -> OpenTelemetry.Instrumentation.AWS.SemanticConventionVersion
+OpenTelemetry.Instrumentation.AWS.SemanticConventionVersion.v1_29_0_Experimental = 2 -> OpenTelemetry.Instrumentation.AWS.SemanticConventionVersion
+OpenTelemetry.Instrumentation.AWS.SemanticConventionVersion.v1_27_0_Experimental = 1 -> OpenTelemetry.Instrumentation.AWS.SemanticConventionVersion
```

Comment:
Is there a reason you want to offer support for 1.27?

Given this instrumentation has not yet received a stable release, I think it would be permissible to update everything to to adhere to 1.29 conventions. Only once you declare the instrumentation stable do you need to pin it to a specific version by default.

Reply:
@alanwest - 

> Is there a reason you want to offer support for 1.27

You're right that we could bump to the latest as part of this PR as the library is not yet stable.  But I wanted to include at least 2 versions of Semantic Convention in this PR so there's a clear pattern established on how to add additional versions moving forward.

Additionally, there was some discussion ([link](https://github.com/open-telemetry/opentelemetry-dotnet-contrib/pull/2367#discussion_r1875510194)) where we decided to default to the Semantic Convention currently in use (~1.27) and then users can opt-in to selecting a higher version.  This way, there's less disruptions to end users.

Thoughts?